src/applications/disability-benefits/2346/sass/form-2346.scss

```diff
@@ -97,6 +103,10 @@ label {
       margin: 1.5rem 0 2.5rem;
     }
 
+    input[type="text"]:disabled {
+      background-color: #f1f1f1;
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/11999/files#r406826151